spec/views/gardens/index.html.haml_spec.rb

```diff
@@ -5,6 +5,7 @@
     controller.stub(:current_user) { nil }
     @owner = FactoryGirl.create(:member)
     @garden = FactoryGirl.create(:garden, :owner => @owner)
+    @finished_planting = FactoryGirl.create(:finished_planting, :garden => @garden)
```

Comment:
Unfortunately, view specs are deprecated. Can you turn this into a feature spec using Capybara instead?